### ISSUE TITLE
Formatting strips 'local' animation modifiers

### ISSUE THREAD

**Issue by lahickey** [UnwrittenFun/svelte-vscode#51]
When ```svelte.plugin.svelte.format.enable``` is set to true (the default), saving a file with animation modifiers:
```
<p transition:fade|local>
    fades in and out only when y changes
</p>
```
results in the modifier (`local` in this case) being stripped off, resulting in this:
```
<p transition:fade>
    fades in and out only when y changes
</p>
```

**Comment by jamesbirtles** [comment]
should be fixed in v0.9.2

**Comment by lahickey** [comment]
Yes, seems to - thanks for the fix
